### ISSUE TITLE
extended.functional -XXgc:enableDoubleMapping  Unknown option: enableDoubleMapping

### ISSUE THREAD

**Issue by pshipton** [eclipse/openj9#8321]
https://ci.eclipse.org/openj9/job/Test_openjdk8_j9_extended.functional_s390x_linux_Personal/69
https://ci.eclipse.org/openj9/job/Test_openjdk11_j9_extended.functional_s390x_linux_Personal/54
SyntheticGCWorkload_DoubleMap_J9_0
```
===============================================
Running test SyntheticGCWorkload_DoubleMap_J9_0 ...
===============================================
SyntheticGCWorkload_DoubleMap_J9_0 Start Time: Wed Jan 15 22:50:24 2020 Epoch Time (ms): 1579128624611
variation: -Xmx2g -Xms2g -Xdump:none -Xgcpolicy:balanced -verbose:gc -XXgc:enableDoubleMapping
JVM_OPTIONS: -Xcompressedrefs -Xmx2g -Xms2g -Xdump:none -Xgcpolicy:balanced -verbose:gc -XXgc:enableDoubleMapping 
JVMJ9GC040E Unknown option: enableDoubleMapping
JVMJ9VM015W Initialization error for library j9gc29(2): Failed to initialize; unable to parse command line
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Comment by pshipton** [comment]
https://github.com/eclipse/openj9/pull/8254
@bragaigor @amicic

**Comment by bragaigor** [comment]
We've just updated the double mapping command line option. It changed from `-XXgc:enableDoubleMapping` to `-XXgc:enableArrayletDoubleMapping`

**Comment by pshipton** [comment]
Seems the testing needs to be updated to match.

**Comment by bragaigor** [comment]
Yes, I'm not sure how to do that but I'll check with @amicic

**Comment by amicic** [comment]
@llxia Please, advise where is the source of test.

**Comment by smlambert** [comment]
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/functional/SyntheticGCWorkload/playlist.xml#L150-L169

**Comment by bragaigor** [comment]
I have the update here: https://github.com/AdoptOpenJDK/openjdk-tests/pull/1560 Is there anything else that we should update?

**Comment by pshipton** [comment]
@smlambert is updating this test going to cause failures in the 0.18.0 release testing?

**Comment by smlambert** [comment]
yes, unless you are using a specific SHA for openjdk-tests repo in 0.18 testing (which I am fairly certain you are not).  Also, we are in a code freeze in the openjdk-tests repo until this release week is finished, so I will not be merging @bragaigor 's PR until the major platforms are completed.

**Comment by pshipton** [comment]
Then I'm going to revert #8254 until the tests can be updated, as it's causing all extended testing to fail. Although I'd prefer to have the Adopt testing updated, and use a SHA for the 0.18.0 testing (or just ignore the failure there). @smlambert would you consider merging this OpenJ9 specific fix?

@DanHeidinga fyi

**Comment by amicic** [comment]
We can just remove the option from the test, since we are enabling it by default. It would solve 0.18 problem in a way that it does not test double mapping, but that's ok since it's not officially available there.

**Comment by bragaigor** [comment]
If it comes down to reverting #8254 I was thinking, we could just create a PR that changes the command line option to what it was previously and that should fix those tests. We'll eventually enable double map by default after this release; at that time, we can update the command line option as well as testing. Not sure how much headache reverting a PR is, that's why I'm suggesting this. @pshipton @smlambert what you think?

**Comment by pshipton** [comment]
@bragaigor it's easy to revert as long as there are no conflicts (which there aren't right now), it's just a button push. If you prefer to open a new PR to temporarily revert the option names, that works as well. I want to avoid all the test failures tonight and over the weekend that are going to occur until we can update the testing.

**Comment by bragaigor** [comment]
I had a branch with this quick fix so just opened the PR here #8326

**Comment by pshipton** [comment]
#8326 is merged so I'll close this. Still remains to update the tests and change the options to the desired names, but I don't think we need this to track it.
